string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,9 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  var wordLowerCased = word.toLowerCase();
+  if (word === wordLowerCased) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
You can also handle this in one line, because conditions will always evaluate to true or false!

`return word === word.toLowerCase()`